core/drivers/rng_pta.c

```diff
@@ -0,0 +1,93 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/*
+ * Copyright (C) 2021, Foundries Limited
+ */
+
+#include <crypto/crypto.h>
+#include <drivers/rng_pta_client.h>
```

Comment:
PTAs APIs are published in **lib/** and exported in OP-TEE OS devkit.
```shell
$ find lib/ -name \*pta\*
lib/libutee/tee_socket_pta.c
lib/libutee/arch/arm/gprof/gprof_pta.h
lib/libutee/arch/arm/gprof/gprof_pta.c
lib/libutee/tee_system_pta.c
lib/libutee/include/pta_invoke_tests.h
lib/libutee/include/pta_system.h
lib/libutee/include/pta_secstor_ta_mgmt.h
lib/libutee/include/pta_gprof.h
lib/libutee/include/pta_benchmark.h
lib/libutee/include/pta_socket.h
lib/libutee/include/scp03_pta.h
lib/libutee/include/pta_device.h
```

I suggest here **lib/libutee/include/pta_rng.h**

Reply:
ok